src/topology/algebra/group.lean

```diff
@@ -940,6 +940,27 @@ local attribute [instance] topological_group.regular_space
 @[to_additive]
 lemma topological_group.t2_space [t1_space G] : t2_space G := regular_space.t2_space G
 
+variables {G}
+
+@[to_additive]
+lemma subgroup.t1_quotient_of_is_closed (S : subgroup G) [S.normal] (hS : is_closed (S : set G)) :
```

Comment:
Any reason why this should not be an instance?
```suggestion
instance subgroup.t1_quotient_of_is_closed (S : subgroup G) [S.normal] [is_closed (S : set G)] :
```
(and likewise below)

Reply:
Oh because I didn't know `is_closed` was a class !